pkg/virt-controller/watch/vm.go

```diff
@@ -324,7 +324,7 @@ func (c *VMController) handleVMRenameRequest(vm *virtv1.VirtualMachine, newName
 	newVM.Status.Conditions = []virtv1.VirtualMachineCondition{
 		{
 			Type:    virtv1.RenameConditionType,
-			Status:  "Success",
+			Status:  k8score.ConditionTrue,
```

Comment:
Please try to avoid unrelated changes in PRs.. even if they are very small (unless they are required for the functionality of this PR)

Reply:
Ok, I will post another PR.